docker-compose.yml

```diff
@@ -0,0 +1,14 @@
+services:
+  mongo:
+    container_name: mongo-on-docker
+    image: mongo:5.0.6
+    environment:
+      MONGO_INITDB_ROOT_USERNAME: "${MONGO_INITDB_ROOT_USERNAME}"
+      MONGO_INITDB_ROOT_PASSWORD: "${MONGO_INITDB_ROOT_PASSWORD}"
+    ports:
+      - "8080:27017"
+    volumes:
+      - ./edulaw-data:/data/db
+
+volumes:
```

Comment:
Volumes tells docker/mongodb where to source our persisted data from. If you create a user, stop the container, and restart it, the user will still exist in our database.